ert3/console/__init__.py

```diff
@@ -66,6 +67,23 @@ def _run_experiment(workspace_root, experiment_name):
     if _experiment_have_run(experiment_root):
         raise ValueError(f"Experiment {experiment_name} have been carried out.")
 
+    coefficients = [
+        (random.gauss(0, 1), random.gauss(0, 1), random.gauss(0, 1))
+        for _ in range(1000)
+    ]
+    data = evaluator(coefficients)
+
+    result = [
+        {
+            "input": {"coefficients": {"a": a, "b": b, "c": c}},
+            "output": {"polynomial_output": output["polynomial_output"]},
+        }
+        for ((a, b, c), output) in zip(coefficients, data)
+    ]
+
+    with open(experiment_root / "data.json", "w") as f:
```

Comment:
May I suggest you write it to `.storage.json` instead of `data.json`. Then you can implement `export` as a copy of `.storage.json` into `data.json`. It is slightly more true to the concept of `run` and `export`.

Reply:
I don't mind changing the name - we just didn't think much of touching this part of the code, as it would likely interfere with whatever changes done in the storage PR

Reply:
I was just thinking that then the export actually have the expected effect :)

Reply:
I misread your intention here. We've updated the run and export now, and I believe this is in agreement with the suggestion.